.github/workflows/CI.yml

```diff
@@ -7,31 +7,39 @@ jobs:
     name: lint
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
-      - uses: actions/cache@v1
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
         with:
-          path: ~/.cache/yarn
-          key: ${{ runner.os }}-yarn-${{ hashFiles(format('{0}{1}', github.workspace, '/yarn.lock')) }}
+          node-version: '16'
+      - name: Cache npm modules
```

Comment:
All the cache blocks can be replaced with `cache: 'npm'` in the setup-node with block

Reply:
@eddiemonge  `cache: 'npm'` doesn't work when no package-lock.json is checked in at the project root.